package.json

```diff
@@ -106,12 +106,13 @@
 	"dependencies": {
 		"@woocommerce/components": "4.0.0",
 		"compare-versions": "3.5.1",
-		"downshift": "3.4.3",
+		"downshift": "3.4.7",
 		"eslint-plugin-woocommerce": "file:bin/eslint-plugin-woocommerce",
 		"gridicons": "3.3.1",
 		"react-number-format": "4.3.1",
+		"serialize-javascript": "2.1.1",
```

Comment:
I'd rather not explicitly do this.  You can use `npm audit fix` and it will update the package-lock.json to the necessary version. This isn't a direct dependency of our own package.